discovery-infra/tests/base_test.py

```diff
@@ -26,6 +26,8 @@ def nodes(self, setup_node_controller):
         controller = setup_node_controller
         nodes = Nodes(controller, env_variables["private_ssh_key_path"])
         nodes.set_correct_boot_order(start_nodes=False)
+        nodes.run_for_all_nodes("reset_cpu_cores")
```

Comment:
this is not something specific to test?
I think this must be part of specific test fixture there is not need for this for all tests

Reply:
maybe we will leave it here anyway? its one place for cleanup, more intuitive. what WDYT?